search-app/docs/api.md

```diff
@@ -1126,6 +1127,85 @@ A couple of parameters used in search expect a date range as input. For example,
 
 Note: ISO 8601 does not allow open-ended time intervals but the CMR API does allow specification of intervals which are open ended on one side. For example, `2000-01-01T10:00:00Z/` and `/2000-01-01T10:00:00Z` are valid ranges.
 
+### <a name="autocomplete-facets"></a> Facet Autocompletion
+
+Auto-completion assistance for building queries. This functionality may be used to help build queries. The facet autocomplete functionality does not search for collections directly. Instead it will return suggestions of facets to help narrow a search by providing a list of available facets to construct a CMR collections search.
```

Comment:
Can you add the description of valid search params for autocompletion search? e.g. the type parameter.